css-backgrounds-4/Overview.bs

```diff
@@ -37,6 +37,171 @@ Introduction</h2>
+			The first and last images touch the edges of the area.
+			If the [=background painting area=] is larger than the background positioning area,
+			then the pattern repeats to fill the background painting area.
+			The value of 'background-position' for this direction is ignored,
+			unless there is not enough space for two copies of the image in this direction,
+			in which case only one image is placed and
+			'background-position' determines its position in this direction.
+
+		<dt><dfn>round</dfn>
+		<dd>
+			The image is repeated in the given direction as often as will fit
+			within the [=background positioning area=].
+			If it doesn't fit a whole number of times,
+			it is rescaled so that it does.
+			See the formula under 'background-size'.
+			If the [=background painting area=] is larger than the background positioning area,
+			then the pattern repeats to fill the background painting area.
+
+		<dt><dfn>no-repeat</dfn>
+		<dd>
+			The image is placed once and not repeated in the given direction.
+	</dl>
+
+	<p>Unless one of the axes is set to ''no-repeat'', the
+	whole background painting area will be tiled, i.e., not just one
+	vertical strip and one horizontal strip.
+
+	<div class="example">
+		<p style="display:none">Example(s):
+		<pre>
+			body {
+				background: white url("pendant.png");
+				background-repeat-y: repeat;
+				background-position: center;
+			}
+		</pre>
+
+		<div class="figure">
+			<p><img src="images/bg-repeat.png" alt="A centered background image, with
+			copies repeated up and down the border, padding and content
+			areas.">
+
+			<p class="caption">The effect of ''repeat'': One copy
+			of the background image is centered, and other copies are put above
+			and below it to make a vertical band behind the element.
+		</div>
+	</div>
+
+	<p>See the section <a href="https://www.w3.org/TR/css-backgrounds-3/#layering">“Layering multiple background
+	images”</a> for how
+	'background-repeat-x',
+	'background-repeat-y',
+	'background-repeat-block',
+	and 'background-repeat-inline'
+	interact with other comma-separated background properties
+	to form each background image layer.
+
+<h3 id="background-repeat" oldids="the-background-repeat">
+Tiling Images Shorthand: the 'background-repeat' property</h3>
+
+	<pre class="propdef">
+	Name: background-repeat
+	Value: <<repeat-style>>#
+	Initial: repeat
+	Applies to: all elements
+	Inherited: no
+	Percentages: N/A
+	Computed value: list, each item a pair of keywords, one per dimension
+	Animation type: discrete
+	</pre>
+
+	<p>This shorthand sets the values for the
+	'background-repeat-x' and 'background-repeat-y' longhand properties.
+	Where
+	<pre class=prod><dfn><<repeat-style>></dfn> = repeat-x | repeat-y | <<repetition>>{1,2}</pre>
+
+	<p>Single values for <<repeat-style>> have the following
+	meanings:
+
+	<dl dfn-type=value dfn-for=background-repeat>
+	<dt><dfn>repeat-x</dfn>
+	<dd>
+	Computes to ''repeat no-repeat''.
```

Comment:
Shorthands don't have a computed value of their own. I would say "Equivalent to `repeat no-repeat`", as in https://drafts.csswg.org/css-flexbox-1/#flex-common